src/gocept/loginuser/bcrypt.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import absolute_import
 from AuthEncoding.compat import b
```

Comment:
Is the compat module part of the API of `AuthEncoding`? Otherwise, we will need to change the code with a new version, which get's rid of the compat module. So we might as well just use the conversion here ourselves.

Reply:
The `b` is an actual function. Let's see how `AuthEncoding` handles dropping Python 2. We can do it the same way later on here. I do not like to copy the code here, if maybe this is not necessary.